cli/pkg/httptest/server.go

```diff
@@ -108,3 +108,53 @@ func (m *MockServer) Close() {
 func (m *MockServer) GetURL() string {
 	return m.server.URL
 }
+
+// func (m *MockServer)
```

Comment:
do we still need this commented out lines?

Reply:
Removed